### DISCUSSION TITLE
problem shutting down pc...

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [sodaliterocks/sodalite#79] [Discussion]
for some reason the system is not shutting down properly, sometimes it loops in the shutdown process and stays at the system logo forever :thinking: 
Could this have something to do with systemd?

**Comment by electricduck** [comment]
During shutdown, if you hit `<Esc>`, you'll be-able to see systemd's output. You'll be-able to see any services it may be getting stuck on and waiting for a timeout on.
